src/SpriteKit/SKNode.cs

```diff
@@ -50,6 +50,27 @@ IEnumerator IEnumerable.GetEnumerator ()
 		{
 			return GetEnumerator ();
 		}
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		public static SKNode Create (string filename, Type [] types, out NSError error)
+		{
+			// Let's fail early.
+			if (filename is null)
```

Comment:
let's use `==`, not `is`, for consistency and not always knowing if `==` was (or not) overloaded.